src/Files.App/Helpers/FilePropertiesHelpers.cs

```diff
@@ -22,35 +23,25 @@ public static async void ShowProperties(IShellPage associatedInstance)
             if (associatedInstance.SlimContentPage.IsItemSelected)
             {
                 if (associatedInstance.SlimContentPage.SelectedItems.Count > 1)
-                {
                     await OpenPropertiesWindowAsync(associatedInstance.SlimContentPage.SelectedItems, associatedInstance);
-                }
                 else
-                {
                     await OpenPropertiesWindowAsync(associatedInstance.SlimContentPage.SelectedItem, associatedInstance);
-                }
             }
             else
             {
-                if (!System.IO.Path.GetPathRoot(associatedInstance.FilesystemViewModel.CurrentFolder.ItemPath)
-                    .Equals(associatedInstance.FilesystemViewModel.CurrentFolder.ItemPath, StringComparison.OrdinalIgnoreCase))
-                {
+                var path = System.IO.Path.GetPathRoot(associatedInstance.FilesystemViewModel.CurrentFolder.ItemPath);
+                if (path is not null && path.Equals(associatedInstance.FilesystemViewModel.CurrentFolder.ItemPath, StringComparison.OrdinalIgnoreCase))
                     await OpenPropertiesWindowAsync(associatedInstance.FilesystemViewModel.CurrentFolder, associatedInstance);
-                }
                 else
-                {
                     await OpenPropertiesWindowAsync(App.DrivesManager.Drives
-                        .SingleOrDefault(x => x.Path.Equals(associatedInstance.FilesystemViewModel.CurrentFolder.ItemPath)), associatedInstance);
-                }
+                        .Single(x => x.Path.Equals(associatedInstance.FilesystemViewModel.CurrentFolder.ItemPath)), associatedInstance);
```

Comment:
Why isn't this `SingleOrDefault`?

Reply:
The difference is that, by using `Single`, an exception is thrown if no item satisfies the conditions (`SingleOrDefault` would return `default`).
In this case, I replaced `SingleOrDefault` with `Single` because the condition is always satisfied by exactly one element.
Let me explain: the only way to enter this branch is by requesting the properties of a drive. To do so, you have to select it, so that drive must exist

Reply:
Sounds reasonable